### DISCUSSION TITLE
I would like to ask if anyone has trained a Chinese text recognition model with crnn? The loss is not decreasing anymore, why the accuracy is always 0?

### DISCUSSION THREAD

**Discussion by htaoruan** [open-mmlab/mmocr#1573] [General]
I would like to ask if anyone has trained a Chinese text recognition model with crnn? The loss is not decreasing anymore, why the accuracy is always 0?

**Comment by htaoruan** [comment]
Dataset labels are error free

**Comment by gaotongxiao** [comment]
Have you taken a look at CRNN's text output? Your dataset size might not be sufficiently large to teach CRNN to use the huge vocabulary base.

**Comment by htaoruan** [comment]
Thank you very much. That is to say, although the dicts I use is set to common Chinese and English characters, my dataset is too small to saturate the crnn model.
